matrix/tasks/glitch/main.py

```diff
@@ -104,9 +104,12 @@ def default_resolver(model, kind, name):
             continue
 
         # Run the specified action on those units
-        rule.log.debug("GLITCHING {}: {}".format(actionf.__name__, action))
+        rule.log.info("GLITCHING {}: {}".format(actionf.__name__, objects))
 
-        await actionf(rule, model, objects, **action)
+        try:
+            await asyncio.wait_for(actionf(rule, model, objects, **action), 30)
```

Comment:
After we've defined success and failure better, as per https://github.com/juju-solutions/matrix/issues/41, we should make this fail, I think, as timeouts are generally indicative of bugs.

... or maybe we have a "test core" config param, and if that is true, then timeouts lead to failure.

In any case, I'm okay with this making it into the codebase as is for now.